packages/neos-ui-sagas/src/UI/ContentCanvas/index.js

```diff
@@ -55,7 +56,23 @@ export function * watchReload() {
             // If the new uri is provided in the action payload, use that
             //
             if (iframeWindow.location.href === currentIframeUrl) {
-                iframeWindow.location.href = uri || iframeWindow.location.href;
+                if (uri) {
+                    const state = yield select();
```

Comment:
as @grebaldi noted, yield is not allowed here and we have to get the state above / or convert it to a for of loop.